packages/student/src/atoms/LinkHOC.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { Link } from "native-base";
+
+const LinkHOC = ({ children, ...rest }) => {
```

Comment:
It will be better to have them as a saperate component for each HOC as there will be a case in future to do some manipulation in data before we get it to the parent component.